README.md

```diff
@@ -135,13 +136,27 @@ git submodule update --remote
 
 ### Testing
 
-The test suite in this project was specifically built to produce consistent results on every run, regardless of when they run or who is running them. This project uses [VCR](https://github.com/dnaeon/go-vcr) to record and replay HTTP requests and responses via "cassettes". When the suite is run, the HTTP requests and responses for each test function will be saved to a cassette if they do not exist already and replayed from this saved file if they do, which saves the need to make live API calls on every test run. If you receive errors about a cassette expiring, delete and re-record the cassette to ensure the data is up-to-date.
+The test suite in this project was specifically built to produce consistent results on every run, regardless of when
```

Comment:
I'd argue that the auto formatting of a line length for markdown makes things less readable since these are just paragraphs, splitting them up oddly like this adds much more lines and because we don't enforce any markdown formatting tools it won't be consistent. I'd suggest we leave this as it was since all of our READMEs are this way